src/ts/component/SearchResultItem/SearchResultItem.tsx

```diff
@@ -0,0 +1,352 @@
+import React from 'react'
+import Col from 'react-bootstrap/Col'
+import Row from 'react-bootstrap/Row'
+import {
+  get,
+  template,
+  uniq
+} from 'lodash-es'
+import { getConfig } from '../../utils/getConfig'
+import TextIcon from '../TextIcon/TextIcon'
+
+function toDateStr(dateTimeStr: string | null): string | null {
+  if (dateTimeStr) return dateTimeStr.split('T')[0]
+
+  return null
+}
+
+function ummTemporalToHuman(umm: object): string | null {
+  const singleDate = get(umm, ['TemporalExtents', 0, 'SingleDateTimes', 0])
+  if (singleDate) return toDateStr(singleDate)
+
+  const rangeDate = get(umm, ['TemporalExtents', 0, 'RangeDateTimes', 0])
+  if (!rangeDate) return null
+
+  const start = toDateStr(rangeDate.BeginningDateTime)
+  const end = toDateStr(rangeDate.EndingDateTime)
+
+  if (!start && !end) return null
+  if (start === end) return start
+  if (!end) return `${start} ongoing`
+
+  return `${start} to ${end}`
+}
+
+interface GeoPoint {
```

Comment:
I feel like all the interfaces should be at the top of the file, instead of between these various util functions. Ideally pull out the utils, but I'd be fine with just grouping similar blocks of code together